src/redux/Topic/saga.js

```diff
@@ -116,19 +146,24 @@ export function* calculateWinningsHandler() {
 
         // return only winning addresses
         if (botWon || qtumWon) {
-          value.push({ address: senderAddress, botWon, qtumWon });
+          withdrawableAddresses.push({
+            type: WithdrawType.winnings,
+            address: vote.voterQAddress,
+            botWon,
+            qtumWon,
+          });
         }
```

Comment:
missing the escrow address i believe?

Reply:
it's not needed. the escrow holder is the AddressManager. As long as the BE knows the topicAddress, the topic has the AddressManager's address so it knows where to withdraw from.